src/App.js

```diff
@@ -19,6 +20,9 @@ class App extends Component {
     const loggedInUser = loggedInUserJson ? JSON.parse(loggedInUserJson) : null;
     this.state = {
```

Comment:
you're using ```this.state = {``` here but on other pages you're using ```state = {``` -- is there a reason this is not consistent with the other pages?